lib/twilio.js

```diff
@@ -34,6 +38,7 @@ async function sendTwilioResponse(response, message) {
 }
 
 module.exports = {
+    isValidTwilioRequest,
     sendTwilioMessage,
-    sendTwilioResponse,
```

Comment:
It's actually good to keep these trailing commas in lists like this that change frequently. It makes the diff easier to understand when looking at pull requests or looking back on commit changes. Here's a little more info: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Reply:
Fixed :)